src/common/KlayTabs.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts" name="KlayTabs">
```

Comment:
I think this component should replace tabs used in Liquidity Remove module.

Reply:
Yes, I planned to do it